src/utils/dev_react_import.js

```diff
@@ -0,0 +1,5 @@
+if(process.env.NODE_ENV === 'development'){
```

Comment:
where did you see this solution?

Reply:
I find this very strange. not sure why this was an issue. please ellaborate.

Reply:
Its to determine which environment you're on. If you're on dev, then we need to import React, because the demo build requires it. 

If you're on production, then we don't want to import React, since its already imported. Thus you can't import it twice.